src/Routes.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Route, Switch } from "react-router-dom";
+import Signup from "./views/SignupPage";
+import ConfirmEmail from "./views/SignupConfirm";
+
+const Routes = () => {
+  return (
+    <Switch>
+      <Route path="/signup" exact component={Signup} />
+      <Route path="/signup/confirmEmail" exact component={ConfirmEmail} />
```

Comment:
I guess, you can tweak your route that way:
```suggestion
      <Route path="/signup/confirm-email" exact component={ConfirmEmail} />
```